server/container/guice/guice-common/src/test/java/org/apache/james/AbstractJamesServerTest.java

```diff
@@ -61,9 +62,28 @@ public void setup() throws Exception {
 
     protected abstract GuiceJamesServer<Id> createJamesServer();
 
+    protected abstract void clean();
```

Comment:
it is supposed the other way around : the children should call the parent method